### ISSUE TITLE
[FEATURE] Add alignment description to heading and paragraph controls

### ISSUE THREAD

**Issue by dingo-d** [infinum/eightshift-frontend-libs#271]
It's a bit tricky to guess what alignment is used for which component in the toolbar in the compound components. The alignment toolbar can be extended to change this. An example in `paragraph-toolbar.js` looks like this

```js
import React from 'react'; // eslint-disable-line no-unused-vars
import { __ } from '@wordpress/i18n';
import { AlignmentToolbar } from '@wordpress/block-editor';
import { alignLeft, alignRight, alignCenter } from '@wordpress/icons';
import { Fragment } from '@wordpress/element';
import { checkAttr } from '@eightshift/frontend-libs/scripts/helpers';
import manifest from './../manifest.json';

const { options } = manifest;

export const ParagraphToolbar = (attributes) => {
	const {
		setAttributes,
		componentName = manifest.componentName,
		paragraphShowControls = true,

		paragraphUse = checkAttr('paragraphUse', attributes, manifest, componentName),

		paragraphAlign = checkAttr('paragraphAlign', attributes, manifest, componentName),

		showParagraphAlign = true,
	} = attributes;

	const alignmentControls = [
		{
			icon: alignLeft,
			title: __( 'Align paragraph left', 'eightshifft-frontend-libs'),
			align: 'left',
		},
		{
			icon: alignCenter,
			title: __( 'Align paragraph center', 'eightshifft-frontend-libs'),
			align: 'center',
		},
		{
			icon: alignRight,
			title: __( 'Align paragraph right', 'eightshifft-frontend-libs'),
			align: 'right',
		},
	];

	if (!paragraphShowControls) {
		return null;
	}

	return (
		<Fragment>
			{paragraphUse &&
				<Fragment>
					{showParagraphAlign &&
						<AlignmentToolbar
							value={paragraphAlign}
							options={options.aligns}
							alignmentControls={alignmentControls}
							label={__('Paragraph alignment', 'eightshifft-frontend-libs')}
							onChange={(value) => setAttributes({ [`${componentName}Align`]: value })}
						/>
					}
				</Fragment>
			}
		</Fragment>
	);
};
```

![Screenshot 2021-02-05 at 12 49 42](https://user-images.githubusercontent.com/8638515/107030250-a419ee80-67b0-11eb-8857-d159b1a1e785.png)

**Comment by iruzevic** [comment]
Let me think about this maybe we can make a more generic solution

**Comment by dingo-d** [comment]
Just a word of advice, setting custom icons is a no-go, because of the way AlignmentToolbar handles icons. 

Also, not every toolbar will need every option, so that's something to think about.

**Comment by goranalkovic-infinum** [comment]
Fixed in #414
